packages/app/src/utils/styledShouldForwardProp.ts

```diff
@@ -0,0 +1,5 @@
+import shouldForwardProp from '@styled-system/should-forward-prop';
+
+type StyledShouldForwardPropType = (prop: string | number) => boolean;
```

Comment:
Once you rename the exported function, this doesn't need the Type suffix. 

```suggestion
type StyledShouldForwardProp = (prop: string | number) => boolean;
```